docs/pf-keys/tutorials/solid_files.rst

```diff
@@ -0,0 +1,111 @@
+********************************************************************************
```

Comment:
I'm OK with this for now but I think this section needs work.   E.G. the term 'patch matrix' is not really described and I don't think this appears in any other PF documentation.   I don't think it is very clear how the labeling is being done.

Reply:
Sure, all of it is mostly a first pass.